frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -1,5 +1,186 @@
 import React from "react";
+import FormThatPrints from "../inputs/form-that-prints.component.js";
+import TextInput from "../inputs/text-input.component.js";
+import Section from "../inputs/section.component.js";
+import GroupSelect from "../inputs/group-select.component.js";
+import Radio from "../inputs/radio.component.js";
+import Checkbox from "../inputs/checkbox.component.js";
+import TextArea from "../inputs/text-area.component.js";
+import TextInputGroup from "../inputs/text-input-group.component.js";
+import {
+  courtTypeOptions,
+  JusticeCourtList,
+  DistrictCourtList
+} from "../form-common-options/form-common-options";
 
-export default function ProofOfCompletedServicd_Web(props) {
-  return <div>Proof of Completed Service Web</div>;
+export default function ProofOfCompletedService_Web({ data }) {
+  return (
+    <FormThatPrints>
+      <Section name="Personal Information">
+        {/* <TextInput dataKey="person.firstName" label={__("First Name")} />
+        <TextInput dataKey="person.middleName" label={_("Middle Name")} />
+        <TextInput dataKey="person.lastName" label={_("Last Name")} />
+        <TextInput dataKey="person.addressStreet" label={_("Street Address")} />
+        <TextInput dataKey="person.addressCity" label={_("City")} />
+        <TextInput dataKey="person.addressState" label={_("State")} />
+        <TextInput dataKey="person.addressZip" label={_("Zip Code")} />
+        <TextInput dataKey="person.phone" label={_("Phone")} />
+        <TextInput dataKey="person.email" label={_("Email")} /> */}
```

Comment:
Why are these commented lines of code here? They look the same as the next lines of code - let's delete these commented lines.

Reply:
Fixed translation labels and removed duplicated commented out code